arangograph/certificates.md

```diff
@@ -0,0 +1,150 @@
+certificate's expiration date. An X.509 certificate can be signed by a
+certificate authority (CA) or self-signed.
+
+ArangoGraph is using:
+- **well known X.509 certificates** created by
+[Let's Encrypt](https://letsencrypt.org/){:target="_blank"}
+- **self-signed X.509 certificates** created by ArangoGraph platform
+
+## Certificate chains
+
+A certificate chain, also called the chain of trust, is a hierarchical structure
+that links together a series of digital certificates. The trust in the chain is
+established by verifying the identity of the issuer of each certificate in the
+chain. The root of the chain is a trusted third-party, such as a certificate
+authority (CA). The CA issues a certificate to an organization, which in turn
+can issue certificates to servers and other entities. 
+
+For example, when you visit a website with an SSL/TLS certificate, the browser
+checks the chain of trust to verify the authenticity of the digital certificate.
+The browser checks to see if the root certificate is trusted, and if it is, it
+trusts the chain of certificates that lead to the end-entity certificate.
+If any of the certificates in the chain are invalid, expired, or revoked, the
+browser will not trust the digital certificate.
+
+## X.509 certificates in ArangoGraph
+
+Each ArangoGraph deployment is accessible on two different port numbers:
+- default port `8529`
+- high port `18529`
+
+Each ArangoGraph Notebook is accessible on two different port numbers:
+- default port `8840`
+- high port `18840`
+
+Metrics are accessible on two different port numbers:
+- default port `8829`
+- high port `18829`
+
+The distinction between these port numbers is in the certificate used for the
+TLS connection.
+
+### Well known X.509 certificates
+
+**Well known X.509 certificates** created by
+[Let's Encrypt](https://letsencrypt.org/){:target="_blank"} are used on the
+default ports, i.e. `8529`.
+
+This type of certificate has a lifetime of 5 years and is rotated automatically.
+It is recommended to use well known certificates, as this eases the access of a
+deployment in your browser.
+
+### Self-signed X.509 certificates
+
+**Self-signed X.509 certificates** are used on the high ports, i.e. `18529`.
+This type of certificate has a lifetime of one year and it is created by the
+ArangoGraph platform. It is also rotated automatically before the expiration
+date.
+
+{% hint 'info' %}
+Unless you switch off the **Use well known certificate** option in the
+certificate generation, both the default and high port serve the same
+self-signed certificate.
+{% endhint %}
+
+### Subject Alternative Name (SAN)
+
+The Subject Alternative Name (SAN) is an extension to the X.509 specification 
+that allows you to specify additional host names for a single SSL certificate.
+
+When using [private endpoints](deployments.html#how-to-create-a-private-endpoint-deployment),
+[notebooks](notebooks.html), and [metrics](monitoring-metrics.html), you can specify 
```

Comment:
remove `[notebooks](notebooks.html), and [metrics](monitoring-metrics.html),` part?

Reply:
done.